requirements.md

```diff
@@ -66,10 +66,49 @@ These are mostly in response to trends we see in issues made
 - Switch between slower HiFi resampler and a fast LoFi one.
 
 ### Needed by Bevy
-Bevy is a widely used game engine build around the ECS model. Because bevy uses ECS it has specific demands from an audio engine.
 
-*TODO*
+Bevy is a popular game engine built around the ECS paradigm.
+Its ECS is highly flexible, so there are many valid ways to integrate
+audio engines. However, idiomatic integrations
+should strive to satisfy three conditions:
 
+1. ECS-driven design.
+2. Broad platform support.
+3. Integration with Bevy's asset system.
+
+#### ECS-driven design
+
+Audio engines are a good fit for ECS-driven design. Playing sounds
+can be modeled by spawning sound entities. Audio tracks and buses
+can be modeled as groups of related entities. Depending on the
+engine, it may also make sense to model effects as entities.
+
+Many audio engines will facilitate ECS-driven integrations out of the box.
+The only hard requirement is that audio types must implement Bevy's 
+[`Component`](https://docs.rs/bevy/latest/bevy/ecs/component/trait.Component.html)
+trait. This can be done directly, through a newtype or other wrapper,
```

Comment:
A reasonably hard requirement is that the types that would be components are Send / Sync, as Bevy is parallel by default and we want these components to be usable in those contexts.

Reply:
Given that audio engine necessarily have to deal with cross-thread communication, this will not be that hard of an extra requirement to meet, I don't think.